addons/web/static/src/legacy/js/views/abstract_renderer_owl.js

```diff
@@ -11,24 +13,22 @@ const FOCUSABLE_ELEMENTS = [
     '[tabindex="0"]'
 ].map((sel) => `:scope ${sel}`).join(', ');
 
-class AbstractRenderer extends owl.Component {
+class AbstractRenderer extends Component {
 
-    constructor() {
-        super(...arguments);
+    setup() {
         // Defines the elements suppressed when in demo data. This must be a list
         // of DOM selectors matching view elements that will:
         // 1. receive the 'o_sample_data_disabled' class (greyd out & no user events)
         // 2. have themselves and any of their focusable children removed from the
         //    tab navigation
         this.sampleDataTargets = [];
-    }
-
-    mounted() {
-        this._suppressFocusableElements();
-    }
 
-    patched() {
-        this._suppressFocusableElements();
+        onMounted(() => {
```

Comment:
you will need to check the classes that inherit from AbstractRenderer, for example MapRenderer